spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/archaius/ArchaiusAutoConfiguration.java

```diff
@@ -165,14 +158,30 @@ protected void configureArchaius(ConfigurableEnvironmentConfiguration envConfig)
 			config.setContainerConfigurationIndex(config
 					.getIndexOfConfiguration(appOverrideConfig));
 
-			ConfigurationManager.install(config);
-		}
-		else {
+			addArchaiusConfiguration(config);
+		} else {
 			// TODO: reinstall ConfigurationManager
 			log.warn("Netflix ConfigurationManager has already been installed, unable to re-install");
 		}
 	}
 
+	private void addArchaiusConfiguration(ConcurrentCompositeConfiguration config) {
```

Comment:
No there is isn't a way to restrict static usage, but almost all of the v1 netflix tools have them and we provide a "spring way" of consuming them.  I'd like to continue that.  Version 2 of the netflix tools move away from statics anyway, so when they do, the user experience here wouldn't have to change.  See https://github.com/Netflix/archaius/pull/216